src/components/orderDetails/OrderDetails.tsx

```diff
@@ -214,7 +217,7 @@ const OrderDetails: React.FC = () => {
           </div>
           <div className="flex justify-between w-full rounded-b-md border-t py-1 px-3 sm:p-0 border-solid border-[#D4D4D4] bg-[#f4f4f4]">
             <span className="w-auto sm:w-1/6">{WMS}</span>
-            <span className="w-auto sm:w-5/6 font-medium">{}</span>
+            <span className="w-auto sm:w-5/6 font-medium"></span>
```

Comment:
why the empty span ?

Reply:
A value will come here which we don't know at this point. So need to keep a placeholder to accommodate that.